packages/perseus/src/interactive2/get-clip-point.ts

```diff
@@ -0,0 +1,29 @@
+import {vector as kvector} from "@khanacademy/kmath";
+
+import type {Coord} from "@khanacademy/perseus";
+
+// Given `coord` and `angle`, find the point where a line extended
+// from `coord` in the direction of `angle` would be clipped by the
+// edge of the graphie canvas. Then draw an arrowhead at that point
+// pointing in the direction of `angle`.
```

Comment:
The part about drawing an arrowhead is not really part of this function, is it?

Reply:
Good catch!